### ISSUE TITLE
Win10: Blank window if split-screen turned on 

### ISSUE THREAD

**Issue by rustdesk** [rustdesk/rustdesk#229]
> Hello, I am a new user of rustdesk, and recently I encountered a tricky problem when using the software, and I am late in solving it, so I took the liberty to ask for your help. The specific problem is as follows.
> 
> After installing rustdesk, it only shows opened in the taskbar, but the desktop does not show any content, only a shadow flashes occasionally. I look forward to your reply, thank you!
> 
> I guess it is because I used the split-screen function of my computer before which caused this error?

**Comment by rustdesk** [comment]
The issue is solved if split-screen turned off

**Comment by TopsDesk** [comment]
you have two computers. 
Computer A - You are connecting from.
Computer B - you are connecting to.

Where did you enabled split screen on A or B?
And where did you turn off the Split screen on A or B?
Could you please write steps you took to cause this problem and and to resolve this problem. All detail will help.

**Comment by rustdesk** [comment]
I believe it is B.

**Comment by rustdesk** [comment]
Sorry, it is A, it has nothing to do with peer connection.

**Comment by rustdesk** [comment]
It is a Sciter compatibility issue, I have proposed it to Sciter author, but they did not reply. Maybe there is work around.

https://sciter.com/forums/topic/win10-blank-window-if-split-screen-turned-on/

**Comment by rustdesk** [comment]
Or we can write a simple tis demo to reproduce the problem, then Sciter author may help us.

**Comment by TopsDesk** [comment]
I downloaded the latest sciter.dll for windows from here
https://github.com/c-smile/sciter-sdk/raw/master/bin.win/x64/sciter.dll
And tested  restdesk. I am not able to reproduce this problem any more. Here are the screenshots.
1. With the menu bar of rustdesk.

2. Full Screen mode.

I am even able to split screen from within RustDesk
Tried with these monitors. 
1. HP Neon HDMI and Display Port.
2. QNIX 27xx with DVI-DUAL Link.
3. Asus Zephyr 15G gaming computer with Nvidia RTX-3070
4. With AMD Radeon 
5. dELL lATITUDE 5520.

We need rebuild the binaries with the latest DLL/Static libraries and have this user test it.
I want to change the top ribbon a pull down on click with a small PULL Arrow. How can make this change? Similar to TeamViewer. Just show a small pull down arrow, Full Screen and the screen buttons 
I will update the issue
